### PR TITLE
Feature flash message

### PR DESCRIPTION
**Pull request by Joltrix**
@warpech @miyconst Is this an okay representation of a flash message?

If you want me to change something, let me know! And sorry for taking so long to complete this issue, have had my hands filled with Palma

### CODE REVIEW

**Comment by warpech** [comment]
@miyconst could you please review?

File: src/KitchenSink/wwwroot/KitchenSink/FlashMessagePage.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <p> And if you combine that with <code>Scheduling.ScheduleTask</code> and <code>Session.ScheduleTask</code> you can make a server side message appear.</p>
+
+        <button class="btn btn-default" value="{{model.ButtonWasClicked$::click}}" onmousedown="++this.value">Trigger message from Server side </button>
+
+        <template is="dom-if" if="{{model.ServerMessage}}">
+            <p class="alert alert-info">{{model.ServerMessage}} </p>
+        </template>
+
+        <p> <br> And by combining <code>&lt;template is="dom-if"&gt;</code> with a click event and a timed function, by using <code> setTimeout </code> you can achieve the same behavior. </p>
+        <button class="btn btn-default" on-click="ButtonClicked"> Trigger message from Client side </button>
```
Comment (miyconst) [inline src/KitchenSink/wwwroot/KitchenSink/FlashMessagePage.html]: Please rename `ButtonClicked` to `showMessage`.

File: src/KitchenSink/wwwroot/KitchenSink/FlashMessagePage.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+<template>
+    <template is="dom-bind">
+        <p class="lead"> Flash Message </p>
+        <p> By pressing the buttons below, you will trigger a message to be displayed for 3 seconds. It is possible to trigger a message both from the server side, and the client side! </p>
+        <p> By using <code>&lt;template is="dom-if"&gt;</code> it is possible to hide or show an element in the HTML depending on the result of the if-statement. </p>
+
+        <p> And if you combine that with <code>Scheduling.ScheduleTask</code> and <code>Session.ScheduleTask</code> you can make a server side message appear.</p>
+
+        <button class="btn btn-default" value="{{model.ButtonWasClicked$::click}}" onmousedown="++this.value">Trigger message from Server side </button>
```
Comment (miyconst) [inline src/KitchenSink/wwwroot/KitchenSink/FlashMessagePage.html]: Please rename `ButtonWasClicked$` to `ShowMessageClick$`.

**Comment by miyconst** [comment]
>@miyconst could you please review?

Just did so. Two tiny adjustments, other than that it looks good. Will have to see whether it actually works after the fixes.

**Comment by Joltrix** [comment]
@miyconst I've changed the name, so if it works for you, feel free to merge :)

**Comment by miyconst** [comment]
@Joltrix I have pushed some changes. Please run a custom TeamCity build and make sure that it passes. Then merge it.

**Comment by warpech** [comment]
> Please rename `ButtonWasClicked$` to `ShowMessageClick$`.

Acutally, our convention is just `ShowMessage$`, right? The same way how we have `Save$`, `Cancel$`.

**Comment by miyconst** [comment]
>Acutally, our convention is just `ShowMessage$`, right? The same way how we have `Save$`, `Cancel$`.

@warpech could you please point me to where the convention is described? Since I kind of fall out of this track.

**Comment by warpech** [comment]
The convention is not described anywhere, so lets agree on it and describe it in the guidelines. I have started a thread about it: https://github.com/StarcounterPrefabs/Guidelines/issues/12

**Comment by warpech** [comment]
Ping @Joltrix

**Comment by Joltrix** [comment]
@warpech Hadn't noticed that the check passed! Going ahead with the merge :)

**Pull request merged**